airflow/contrib/utils/gcp_credentials_provider.py

```diff
@@ -99,6 +102,44 @@ def get_credentials_and_project_id(
 
     return credentials, project_id
 
+def get_gce_jwt_token():
```

Comment:
Also can we call this `get_sift_gce_jwt_token`? Maybe there is a way to make this more generic, but I'm not sure it's worth the effort.

Reply:
updated name.